lib/ansible/module_utils/eos.py

```diff
@@ -419,16 +425,17 @@ def load_config(self, config, commit=False, replace=False):
 
         return result
 
-is_json = lambda x: str(x).endswith('| json')
-is_text = lambda x: not str(x).endswith('| json')
 
-supports_sessions = lambda x: get_connection(module).supports_sessions
+def is_json(cmd):
+    return str(cmd).endswith('| json')
```

Comment:
It's better to use to_native() than to use str() as you can control the error handling and encoding with to_native.  So something like:
``` python
from ansible.module_utils._text import to_native
[...]
return to_native(cmd, errors='surrogate_or_strict').endswith('| json')
```

Notes: I'm not sure whether raising an error is the appropriate behaviour here as I haven't looked deeper into the code.  Perhaps you want the text <=> bytes conversion to raise an error but then catch it in is_json() and return False?  (But that will tend to obscure that you have bad data if you aren't expecting the conversion to ever fail... You'll have to decide what the correct strategy is but you can ask me for help implementing that if you need to).